### PR TITLE
Collect analytic count

### PR DESCRIPTION
**Pull request by carmenlau**
refs #1472 

require #1505

Done:
- Active user count
- signup and login page view count

### CODE REVIEW

File: pkg/lib/analytic/provider.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package analytic
+
+type CounterStore interface {
+	TrackActiveUser(userID string) error
+}
+
+type Provider struct {
```
Comment (louischan-oursky) [inline pkg/lib/analytic/provider.go:7]: We have switched to name this kind of struct Service.

File: pkg/lib/analytic/store_redis.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+				return err
+			}
+		}
+		return nil
+	})
+	return
+}
+
+func monthlyActiveUserCount(appID config.AppID, year int, month int) string {
+	return fmt.Sprintf("app:%s:monthly-active-user:%d:%d", appID, year, month)
```
Comment (louischan-oursky) [inline pkg/lib/analytic/store_redis.go:56]: Format the month as `2021-09`.

File: pkg/lib/analytic/store_redis.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+func weeklyActiveUserCount(appID config.AppID, year int, week int) string {
+	return fmt.Sprintf("app:%s:weekly-active-user:%d:%d", appID, year, week)
```
Comment (louischan-oursky) [inline pkg/lib/analytic/store_redis.go:60]: Format the week as `2021W40`

**Comment by carmenlau** [comment]
Updated!

**Pull request merged**